client/src/SongDescription.jsx

```diff
@@ -17,7 +17,11 @@ class SongDescription extends React.Component {
   updateDescription(data) {
     console.log('Here is the data sent to updateDescription: ', data);
     this.setState({
-      description: data.description
+      // mongo implementation
+      // description: data.description
```

Comment:
Are you going to return to the mongo implementation for more work? If not, I think you can probably take this cruft out.